ocaml/testsuite/tests/typing-layouts/basics_alpha.ml

```diff
@@ -1801,3 +1801,21 @@ Error: This type signature for foo33 is not a value type.
 (* Test 34: Layout clash in polymorphic record type *)
 
 (* tested elsewhere *)
+
+(****************************************************)
+(* Test 35: unannotated type parameter defaults to layout value *)
```

Comment:
When adding a new test to "alpha", please add stubs to "beta" and the regular test.